CHANGELOG.md

```diff
@@ -13,7 +13,7 @@
 * rhel82 FIPS ([#200](https://github.com/mesosphere/konvoy-image-builder/issues/200)) ([1688a02](https://github.com/mesosphere/konvoy-image-builder/commit/1688a028fc8a350eb04c3bd1355f40c06b90a2f4))
 
 
-### Bug Fixes
+### Bugfixes
```

Comment:
making linter happy. 
```
/github/workspace/CHANGELOG.md
  16:5   ✓ error  Incorrect usage of the term: “Bug Fixes”, use “Bugfixes” instead  terminology
  30:37  ✓ error  Incorrect usage of the term: “repo”, use “repository” instead     terminology
```
however since this file is auto generated and auto merged, we have to find out if there is any other way to fix/ignore this.